perses/tests/test_repex.py

```diff
@@ -188,5 +185,218 @@ def test_RESTCapableHybridTopologyFactory_repex_charge_mutation():
+            host_filename = resource_filename("perses", os.path.join("data", "host-guest", "cb7.sybyl.mol2"))
+            fe_setup = RelativeFEPSetup(
+                ligand_input=[guest_1_filename, guest_2_filename],
+                receptor_mol2_filename=host_filename,
+                old_ligand_index=ligand_A_index,
+                new_ligand_index=ligand_B_index,
+                forcefield_files=["amber14/tip3p.xml"],
+                small_molecule_forcefield="gaff-2.11",
+                small_molecule_parameters_cache=resource_filename("perses", os.path.join("data", "host-guest", "cache.json")),
+                phases=phases,
+                solvent_padding=1.7 * unit.nanometers)
+
+            for phase in phases:
+                # Generate htf
+                topology_proposal = fe_setup.solvent_topology_proposal if phase == 'solvent' else fe_setup.complex_topology_proposal
+                old_positions = fe_setup.solvent_old_positions if phase == 'solvent' else fe_setup.complex_old_positions
+                new_positions = fe_setup.solvent_new_positions if phase == 'solvent' else fe_setup.complex_new_positions
+                htf = RESTCapableHybridTopologyFactory(
+                    topology_proposal=topology_proposal,
+                    current_positions=old_positions,
+                    new_positions=new_positions
+                )
+
+                # Make sure LRC is set correctly
+                hybrid_system = htf.hybrid_system
+                force_dict = {force.getName(): index for index, force in enumerate(hybrid_system.getForces())}
+                custom_force = hybrid_system.getForce(force_dict['CustomNonbondedForce_sterics'])
+                nonbonded_force = hybrid_system.getForce(force_dict['NonbondedForce_sterics'])
+                custom_force.setUseLongRangeCorrection(False)
+                nonbonded_force.setUseDispersionCorrection(True)
+
+                # Create a reporter
+                selection = "not water"
+                reporter_file = os.path.join(temp_dir, f"{ligand_A_index}_{ligand_B_index}_{phase}.nc")
+                reporter = MultiStateReporter(
+                    reporter_file,
+                    analysis_particle_indices=htf.hybrid_topology.select(selection),
+                    checkpoint_interval=10)
+
+                # Build the hybrid repex sampler
+                sampler = HybridRepexSampler(
+                    mcmc_moves=mcmc.LangevinDynamicsMove(
+                        timestep=4.0 * unit.femtoseconds,
+                        collision_rate=1.0 / unit.picosecond,
+                        n_steps=50,
+                        reassign_velocities=False,
+                        n_restart_attempts=20,
+                        constraint_tolerance=1e-06),
+                    replica_mixing_scheme='swap-all',
+                    hybrid_factory=htf,
+                    online_analysis_interval=10)
```

Comment:
Should `online_analysis_interval` be on here? And should it be so frequent?

Reply:
As far as I can tell, we don't really need it, I think we can just leave it as the default value of 200.